session_test.go

```diff
@@ -1310,6 +1311,26 @@ var _ = Describe("Session", func() {
 		close(done)
 	})
 
+	Context("keep-alives", func() {
```

Comment:
Replace the test case with:

```go
		It("sends a ping packet", func() {
			sess.config.Keepalive = true
			sess.lastNetworkActivityTime = time.Now().Add(-(sess.idleTimeout() / 2))
			go sess.run()
			defer sess.Close(nil)
			time.Sleep(60 * time.Millisecond)
			Eventually(func() [][]byte { return mconn.written }).ShouldNot(BeEmpty())
			Eventually(func() byte {
				// -12 because of the crypto tag. This should be 7 (the frame id for a ping frame).
				s := mconn.written[0]
				return s[len(s)-12-1]
			}).Should(Equal(byte(0x07)))
		})
```